src/plasmapy/simulation/particle_tracker/particle_tracker.py

```diff
@@ -118,10 +138,18 @@ def __init__(
         save_routine: AbstractSaveRoutine | None = None,
         dt=None,
         dt_range=None,
+        relativistic_beta_threshold=0.01,
         field_weighting="volume averaged",
         req_quantities=None,
         verbose=True,
     ) -> None:
+        # By default, set the integrator to the explicit Boris push
+        # The `_push()` method may change this to a relativistic integrator if the energies of the particles
+        # exceed the threshold specified by `relativistic_beta_threshold`
+        self._integrator = _INTEGRATORS["explicit_boris"]
+        self._beta_threshold = relativistic_beta_threshold
+        self._is_relativistic = False
```

Comment:
If you do make the integrator a class, then `_is_relativistic can just be a property 

def _is_relativistic(self):
return self._integrator.relativistic